src/winconpty/ft_pty/ConPtyTests.cpp

```diff
@@ -22,44 +22,48 @@ class ConPtyTests
     TEST_METHOD(DiesOnClose);
 };
 
-HRESULT _CreatePseudoConsole(const COORD size,
-                             const HANDLE hInput,
-                             const HANDLE hOutput,
-                             const DWORD dwFlags,
-                             _Inout_ PseudoConsole* pPty)
+static HRESULT _CreatePseudoConsole(const COORD size,
+                                    const HANDLE hInput,
+                                    const HANDLE hOutput,
+                                    const DWORD dwFlags,
+                                    _Inout_ PseudoConsole* pPty)
 {
     return _CreatePseudoConsole(INVALID_HANDLE_VALUE, size, hInput, hOutput, dwFlags, pPty);
 }
 
-HRESULT AttachPseudoConsole(HPCON hPC, LPPROC_THREAD_ATTRIBUTE_LIST lpAttributeList)
+static HRESULT AttachPseudoConsole(HPCON hPC, std::wstring& command, PROCESS_INFORMATION* ppi)

```

Comment:
o_O why'd you refactor this out?

Reply:
sorry, why'd you refactor this *back in*?

Reply:
ah, code duplication. Ok, sure

Reply:
IIRC the problem here was the "you can't pass string literals as mutable `wchar_t*` pointers" issue, but fixing it in the existing code was a hassle. So I just refactored the test real quick. As it turns out I was able to remove a lot of code duplication too.